### ISSUE TITLE
Usage statistics of a newly added step should be displayed

### ISSUE THREAD

**Issue by sguptatw** [getgauge/gauge-java#207]
**Expected behavior**
Usage statistics of a newly added step should be displayed

**Actual behavior**
Usage statistics of a newly added step is not  displayed

**Steps to replicate**
* Create a new `gauge-java` project
* Add a new step
* Give the step implementation in an existing file. 
> The usage statistics are not displayed even after the refresh is complete

![newstep](https://user-images.githubusercontent.com/6310197/49627346-5bb1eb00-fa04-11e8-9084-8b3c83349cb5.png)

**Note**
The statistics is displayed only after switching between spec and implementation file.

**Version**
```
Gauge version: 1.0.4.nightly-2018-12-03
Commit Hash: 62a48e5

Plugins
-------
java (0.7.0.nightly-2018-12-04)
```

> Blocks https://github.com/getgauge/gauge-vscode/issues/83

**Comment by sguptatw** [comment]
[Blocker] Latest nightly available is 0.7.0.nightly-2018-12-04

**Comment by sguptatw** [comment]
Working as expected on version
```
0.7.0.nightly-2018-12-10
```
